SimplenoteUITests/SimplenoteUITestsSearch.swift

```diff
@@ -41,26 +60,26 @@ class SimplenoteUISmokeTestsSearch: XCTestCase {
 
         trackStep()
         NoteList.openAllNotes()
-        NoteListAssert.notesExist(names: [godzillaNoteName, kingKongNoteName, mechagodzillaNoteName, diacriticNoteName])
+        NoteListAssert.notesExist(allNotes)
```

Comment:
I'd argue that not only this is easier to type and read because there are less characters involved, but it's less cognitive demanding because the fact that we're verifying that all notes exist in the list is explicit here. In the previous version, a reader would have had to deduce from the context and by reading the top of the test file that those four notes are all the notes there are.